staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -780,6 +780,10 @@ type KubeletConfiguration struct {
 	// Default: true
 	// +optional
 	RegisterNode *bool `json:"registerNode,omitempty"`
+	// localStorageCapacityIsolation enables local storage capacity isolation.
```

Comment:
Also document why one might want to change this.

Reply:
added the document. PTAL.